.github/workflows/ci.canary.yml

```diff
@@ -235,6 +235,52 @@ jobs:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPMJS_TOKEN: ${{ secrets.NPMJS_TOKEN }}
 
+      - name: Build HTML documentation of SDK-s
+        run: |
+          yarn --cwd packages/sdk-core doc:html
+          yarn --cwd packages/sdk-redux doc:html
+
+      - name: Get sdk-core package.json version
+        id: sdk-core-version
+        uses: martinbeentjes/npm-get-version-action@master
```

Comment:
For the sake of supply chain safety, I'd rather we use  `jq .version packages/sdk-core/package.json".

The problem of using a untrusted action from an arbitrary branch/tag is that, if their code base is compromised, the action might be able to steal github secrets exported.

For example, we should really remove 

```
env:
  GITHUB_TOKEN: ${{ secrets.CI_GITHUB_TOKEN }}
```

From top-level of this workflow, external actions could have unlimited access to it.

Let's fix it.